src/BizHawk.Emulation.Cores/Consoles/Nintendo/Gameboy/GambatteLink.cs

```diff
@@ -160,6 +174,31 @@ private ControllerDefinition CreateControllerDefinition()
 			return ret.MakeImmutable();
 		}
 
+		private bool isSgb(int i)
+		{
+			return _linkedCores[i].IsSgb;
+		}
+
+
+		public bool isAnySgb { get; set; }
+
+		private bool showBorder(int i)
+		{
+			return isSgb(i) && _settings._linkedSettings[i].ShowBorder;
+		}
+
+		public bool showAnyBorder()
+		{
+			for (int i = 0; i < _numCores; i++)

```

Comment:
A more idiomatic way to do this in C#/LINQ:
```c#
int[] coreIndices = Enumerable.Range(0, _numCores).ToArray()
return coreIndices.Any(core => showBorder(core))
```
There may be other places in this code where you could use more idiomatic C# like this but this is just an example

Reply:
You don't need all the allocations that `ToArray` brings: `public bool showAnyBorder() => Enumerable.Range(0, _numCores).Any(showBorder);`